src/views/Dashboard.vue

```diff
@@ -32,7 +32,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           hide-default-header
           id="dashboard-workflows"
         >
-          <v-progress-linear slot="progress" color="grey" indeterminate></v-progress-linear>
```

Comment:
Undocumented change?

Reply:
Removed because a) I've never seen it while using the UI and b) I think the only difference to the default progress bar is the colour, not really worth defining a custom one imho